packages/es-dev-server/src/create-middlewares.ts

```diff
@@ -197,6 +199,10 @@ export function createMiddlewares(config: ParsedConfig, fileWatcher = chokidar.w
   middlewares.push(createPluginMimeTypeMiddleware({ plugins }));
   middlewares.push(createPluginServeMiddlware({ plugins }));
 
+  if (cors) {
```

Comment:
I think we should move this above the `  middlewares.push(createPluginServeMiddlware({ plugins }));` so that we also allow CORS for files served from plugins.

Reply:
I've added an entry in README.md and moved the code one line before